scripting/get5/util.sp

```diff
@@ -422,6 +421,26 @@ stock bool RemoveStringFromArray(ArrayList list, const char[] str) {
   return false;
 }
 
+// Because KeyValue cannot write empty strings, we use this to consistently read empty strings and replace
+// our empty-string-placeholder with actual empty string.
+stock bool ReadEmptyStringInsteadOfPlaceholder(const KeyValues kv, char[] buffer, const int bufferSize) {
+  kv.GetString(NULL_STRING, buffer, bufferSize);
+  if (StrEqual(KEYVALUE_STRING_PLACEHOLDER, buffer)) {
+    Format(buffer, bufferSize, "");
+    return true;
+  }
+  return false;
+}
+
+stock bool WritePlaceholderInsteadOfEmptyString(const KeyValues kv, char[] buffer, const int bufferSize) {
+  kv.GetString(NULL_STRING, buffer, bufferSize);
```

Comment:
Technically this buffer could be inside the function and the arguments could be removed, but since it's always being used in a loop, I let it be a parameter so we don't redefine a `char[]` array each time for no reason.